lib/utils.js

```diff
@@ -1,14 +1,28 @@
 'use strict';
 
-function getMasksFromCommentBody(commentBody) {
+const FuzzyMatching = require('fuzzy-matching');
```

Comment:
Do you think about using https://github.com/sindresorhus/leven or https://github.com/NaturalNode/natural to instead?